charts/fybrik/values.yaml

```diff
@@ -215,8 +215,7 @@ manager:
   solver:
     # image of the container with solver binary and libs
     # when specified, the solver will be deployed in the manager pod
-    image: "optimizer"
-    imagePullPolicy: "Always"
+    image: "ghcr.io/fybrik/optimizer:0.1"
```

Comment:
i think its better to use a major.minor.patch format for the tag following [semantic versioning](https://semver.org/)

Reply:
Changed to 0.0.1